docs/modules/getting_started/pages/flow-deep-dive.adoc

```diff
@@ -3,20 +3,20 @@
 = Flow Deep-Dive Dashboard
 
 The {page-component-title} plugin includes a pre-designed flow deep-dive dashboard that displays flow data.
-Access this dashboard from the home page or from the *Dashboards>Manage* menu.
+Access this dashboard from the home page or from the menu:Dashboards[Manage] menu.
```

Comment:
Eventually we should update to use this syntax throughout, but let's not do that now. It can be part of the review of the entire Helm documentation.